File: iOS/KVODemo/KVODemo/ViewController.m
//
//  ViewController.m
//  KVODemo
//
//  Created by yuecheng on 2019/8/9.
//  Copyright © 2019 NanoSparrow. All rights reserved.
//

#import "ViewController.h"
#import "NASPerson.h"

@interface ViewController ()

@property (nonatomic, strong) NASPerson *person;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    self.person = [[NASPerson alloc] init];
    NSLog(@"%s", object_getClassName(self.person));
    [self.person addObserver:self forKeyPath:@"name" options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld context:nil];
    NSLog(@"%s", object_getClassName(self.person));
}


@end

File: iOS/SocketDemo/SocketDemo/SocketDemo/ViewController.swift
//
//  ViewController.swift
//  SocketDemo
//
//  Created by yuecheng on 2019/8/7.
//  Copyright © 2019 NanoSparrow. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var streamManager = NASStreamManager.init()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.streamManager.connectWithServer(host: "192.168.2.1", port: 65432)
        // Do any additional setup after loading the view.
    }


}

